files/__addonLocation__/rollup.config.js

```diff
@@ -45,5 +46,13 @@ export default {
 
     // Remove leftover build artifacts when starting a new build.
     addon.clean(),
+
+    // Copy Readme and License into published package
+    copy({
```

Comment:
would using symlinks be better?
like, have these live in the addon itself and symlink out to the root?

Reply:
For now I just followed what Ed has been doing in his addon, which is also basically what has been discussed in https://github.com/embroider-build/embroider/pull/1188...

So git seems to support symlinks, but apparently npm does not. I just tried what happens when you do `npm pack` with a package that has a symlink. And neither the actual file (which is what we need) nor the symlink itself makes it into the tarball!

Reply:
@simonihmig correct, symlinks in the npm is the reason why it took some time and there are workarounds none of which seem to be really straightforward.

what is proposed here seem totally legit IMO.

Reply:
yeah, I think we'd need to create the symlinks as part of the addon creation

Reply:
> I think we'd need to create the symlinks as part of the addon creation

No, that won't help. That's what I tried:

```bash
cd my-addon
ln -s ../README.md
npm pack
```

Result: no Readme in tarball!

Reply:
oh, that's the reverse of what I'm suggesting

```bash
# readme is at my-addon/README.md
ln -s ./README.md my-addon/README.md
```

Reply:
Ok, I see. 

Seems this is what ember-animated is doing: https://github.com/ember-animation/ember-animated

The Github UI seems to pick up the symlinked Readme correctly in the default view. But clicking on a symlink does not "follow" the link, but displays the link path: https://github.com/ember-animation/ember-animated/blob/master/README.md. Ok enough I guess, but don't really love it...

@ef4 @SergeAstapov seems you have been involved with both approaches, what are your thoughts?